Gemfile

```diff
@@ -5,6 +5,6 @@ source 'https://rubygems.org'
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
 # Synchronize with https://pages.github.com/versions
-ruby '>=2.5.8'
+ruby '>=2.5.1'
```

Comment:
Why did you change the Gemfile, especially to lower it to a less recent version?

Reply:
It was a temporary fix for me to look at the HTML, it seems the ruby version requirement was not working for my local Linux package, versions. Please ignore this change, this was just for my development branch.